### PR TITLE
Added code for init command

### PR DESCRIPTION
**Pull request by PrathamGupta423**

### CODE REVIEW

File: commands/init.py
Comment (SidZRed) [inline commands/init.py]: Can move outside the loop.

**Comment by DhruvaKashyap** [comment]
I see you are using the os library; could you explore using pathlib?
It may save some code, but one would need to explore its efficiency.

**Comment by PrathamGupta423** [comment]
> I see you are using the os library; could you explore using pathlib? It may save some code, but one would need to explore its efficiency.

I have refactored it for use with the pathlib library; since this is only used in the initial setup and only creates ~3 folders, I don't think efficiency is a major issue.

**Review by kevalpithadiya — APPROVED** [review]
LGTM

Except that this is the first time I have seen `pathlib` being using and I find the syntax to be very cursed.

**Comment by PrathamGupta423** [comment]
@kevalpithadiya can you make this code compatible with recent changes to command handler

**Comment by kevalpithadiya** [comment]
Yes that wouldn't be that big of a deal, can be simply handled by resolving the merge conflict.
But I suggest you directly merge this PR to `main` branch once the PR for command handler is merged.

**Comment by DhruvaKashyap** [comment]
> LGTM
> 
> Except that this is the first time I have seen `pathlib` being using and I find the syntax to be very cursed.

Congrats on your first LGTM.

**Comment by DhruvaKashyap** [comment]
@PrathamGupta423, @kevalpithadiya, please take a look at the merge and let me know if its fine

**Comment by PrathamGupta423** [comment]
> @PrathamGupta423, @kevalpithadiya, please take a look at the merge and let me know if its fine

LGTM, can be merged now

**Comment by kevalpithadiya** [comment]
Yep, looks good to me as well.

**Pull request merged**
